### ISSUE TITLE
Stress rig commands: print to genie python log

### ISSUE THREAD

**Issue by FreddieAkeroyd** [ISISComputingGroup/IBEX#2647]
As a supporter I would like the stress rig commands to log useful information to the genie python log to help with debugging. It should log commands it is executing, and also values of any stress/strain/position  state it saves.

**Comment by GDH-ISIS** [comment]
For easy diagnosis and visibility, would it not be simple and pertinent to alter the stress rig configurations to also record the absolute stress set point, position set point, and strain set point. The parameters could then be viewed by us and the instrument scientists in the log files.

**Comment by FreddieAkeroyd** [comment]
I would probably add them to the instrument archive instead, if the scientists wished we could create blocks for them too.

**Comment by GDH-ISIS** [comment]
Agreed Freddie. I want it to go one step further so they have visibility in the analysis of their data also rather than be tied to IBEX.
